### PR TITLE
Add Sauron; update dev1 environment; update raptor for local development

### PR DESCRIPTION
**Pull request by c-m-duncan**
Adds Sauron to deployment. Used to keep dev environment up to date with the latest versions of smartcitiesdata applications.

Note, manual addition of sauron public access token to github secrets is required.

Also adds envs directory, allowing for unique chart values for each environment. Should likely be split into separate repo(s) in the future.

### CODE REVIEW

File: .github/workflows/teardown_urbanos.yml [hunk 1]
```diff
@@ -37,8 +37,9 @@ jobs:
           mode: auto-approve-destroy
           commentOnPr: false
         env:
-          # ensure edits to these are added to the "teardown_urbanos" workflow as well
+          # ensure edits to these are added to the "deploy_urbanos" workflow as well
```
Comment (nicholas-goodwin) [inline .github/workflows/teardown_urbanos.yml:40]: still feel like this is still teardown_urbanos, due to still being in the teardown_urbanos yml?

File: .github/workflows/teardown_urbanos.yml [hunk 2]
```diff
@@ -54,3 +55,5 @@ jobs:
           JUNO_RAPTOR_AUTH0_CLIENT_SECRET: ${{secrets.JUNO_RAPTOR_AUTH0_CLIENT_SECRET}}
           JUNO_RESOURCE_PREFIX: ${{secrets.JUNO_RESOURCE_PREFIX}}
           JUNO_USE_STAGING_LETS_ENCRYPT: does_not_matter
+          JUNO_ENABLE_SAURON: does_not_matter
```
[the remark this answers is not in the document]
Comment (c-m-duncan) [reply .github/workflows/teardown_urbanos.yml:58]: That variable determines if sauron gets stood up in the setup_urbanos.yml file. The variable is still over here in the teardown file for consistency, but the value isn't used. It just has to exist 🤷

File: scripts/initialize_andi_req_data/create_org_one.json [hunk 3]
```diff
@@ -8,4 +8,4 @@
   "orgName": "urbanos_demo_org",
   "orgTitle": "UrbanOS Demo Org",
   "version": "0.1"
-}
+}
```
Comment (nicholas-goodwin) [inline scripts/initialize_andi_req_data/create_org_one.json:11]: mmm, yes, very good change here /j (I figure this is a formatting change, but funny nonetheless)

File: src/helm.ts [hunk 4]
```diff
@@ -46,6 +46,24 @@ export const installUrbanOS = (
...
   });
 
+export const installSauron = (
+  classRef: TerraformStack,
+  dependsOn: DependsOn
+) =>
+  new Release(classRef, "SauronHelmRelease", {
+    name: "sauron",
+    chart: "sauron",
+    version: "0.0.17",
```
Comment (nicholas-goodwin) [inline src/helm.ts:56]: here has Sauron at 0.0.17, whereas sauron_values.yaml has 0.0.8. Just making sure all is good there and hopefully consistent

**Review by nicholas-goodwin — APPROVED** [review]

File: src/helm.ts (hunk 4 above)
Comment (c-m-duncan) [reply src/helm.ts:56]: Good catch 👍

**Pull request merged**